lib/index.js

```diff
@@ -99,6 +99,21 @@ module.exports = {
                 });
             }
 
+            if (route.security) {
+                var securitySchemes = Object.keys(route.security);
+                Assert.ok(securitySchemes.length < 2, 'HAPI supports 1 authentication strategy per route.');
```

Comment:
That's not actually true? You can configure multiple strategies through config.auth.strategies.

http://hapijs.com/tutorials/auth#route-configuration

Don't think that's going to cause us any issues but suspect if it was to be merged back into the original module they might want to support more cases/configuration.